rfps/privacy-enhancement-polkadot-extension.md

```diff
@@ -0,0 +1,34 @@
+# Privacy Enhancement for Polkadot Extension
+
+* **Status:** Open
+* **Proposer:** jonasW3F
+* **Your Project(s):** [optional]: N/A
+* **Projects you think this work could be useful for** [optional]: https://github.com/polkadot-js/extension
+* **Teams/People that could deliver the RFP** [optional]: N/A
+
+## Project Description :page_facing_up: 
+
+An increasing number of websites require access to the Polkadot extension with a growing ecosystem. By allowing access to the extension, websites (naturally) can see the addresses that are contained in the extension. This imposes a big risk to privacy, because malicious actors could create lists about which addresses belong to one entity and, in the worst case, even link it to real-world identities (if at least one address is linked to an on-chain identity). A feature to prevent this is currently the "hide" button on single addresses, which prevent websites from seeing that address. However, it is currently cumbersome to handle that feature. The idea of this RFP is to extend on that feature and include a few QOL functionalities to make it easier for users to protect their privacy.
```

Comment:
I actually don't have a 'hide' button, which browser are you using?

Reply:
Every account has an eye symbol on the right side, which you can click. It's at least on Chrome and Firefox. Unless your extension is really outdated. ;)

Reply:
yes, you should be able to hide individual accounts already